README.md

```diff
@@ -10,43 +10,43 @@ In this lesson, we'll walk through the steps you need to take to solve a lab on
 
 To start working on a lab on Learn, you need get it opened up on your computer. There are a couple different ways to do this:
 
-1. The first, and simplest, is to just click the `Open` button in the toolbar at the top of a lab on Learn—don't try to do this just yet, you'll do this in the next lesson. When you click Open, you'll notice a Terminal window briefly pop up, then your text editor will open with the contents of the lab's directory. You're now ready to start coding to solve the lab. 
-2. Alternatively, from a terminal window, you can type the `learn open` command which will open up your current lab 
+1. The first, and simplest, is to just click the `Open` button in the toolbar at the top of a lab on Learn—don't try to do this just yet, you'll do this in the next lesson. When you click Open, you'll notice a Terminal window briefly pop up, then your text editor will open with the contents of the lab's directory. You're now ready to start writing code to solve the lab.
+2. Alternatively, from a Terminal window, you can type the `learn open` command which will open up your current lab
 
 You'll notice that on Learn, the light labelled "Fork" will turn green after you've opened up a lab.
 
-So what's going on here? Behind the scenes, the Learn CLI is (1) forking this lab's repository on Github (2) running `git clone` to clone the contents down to your computer (3) running `cd` to change directory into the lab's directory then (4) opening your text editor so you can get to work. 
+So what's going on here? Behind the scenes, the Learn CLI is (1) forking this lab's repository on Github (2) running `git clone` to clone the contents down to your computer (3) running `cd` to change directory into the lab's directory then (4) opening your text editor so you can get to work.
```

Comment:
i am assuming that there is a short lesson ahead of this one that introduces some of the basic git stuff. we'll need to read it in context to know for sure.